### PR TITLE
PEP 609: Clarify expectations around PEP sponsors

### PR DESCRIPTION
**Pull request by pradyunsg**
Title says it all. :)

/cc @di as the co-author on this PEP

### CODE REVIEW

File: pep-0609.rst [hunk 1]
```diff
@@ -164,10 +164,11 @@ Python Steering Council has the final say in the acceptance of these
...
 It is expected (but not required) that the Python Steering Council
-delegates the authority to approve or reject PEPs to individuals
-within the PyPA community. At the time of writing (November 2019), the
-Python Steering Council has `standing delegations`_ for currently
-active packaging interoperability specifications.
+would delegate authority to sponsor and/or approve/reject PEPs related
+to packaging interoperability specifications, to individuals within the
+PyPA community. At the time of writing (June 2020), the Python Steering
+Council has `standing delegations`_ for currently active packaging
+interoperability specifications.
```
Comment (pfmoore) [inline pep-0609.rst:171]: The current standing delegations are for approval, not sponsoring. I don't believe there are any standing delegations for sponsors, these have to be Python core developers as per the normal PEP process.

Comment (di) [reply pep-0609.rst:171]: In https://discuss.python.org/t/pep-609-pypa-governance/2619/12 @ncoghlan said:

> The status quo is really that additional sponsors can be listed in the standing delegation document alongside the standing BDFL-Delegates. Those standing delegates would also be eligible to sponsor PEPs.

I take this to mean that anyone in the standing delegation for approval can also be a sponsor, regardless of their Python core developer status.

Comment (pradyunsg) [reply pep-0609.rst:171]: I take Nick's comment (which is from back when he was on the SC) to also mean that the SC may add individuals to the standing delegations, who can be sponsors for (only) packaging-related PEPs.

Comment (pradyunsg) [reply pep-0609.rst:171]: I take Nick's comment (which is from back when he was on the SC) to also mean that the SC may add individuals to the standing delegations, who can be sponsors for (only) packaging-related PEPs.

Comment (pfmoore) [reply pep-0609.rst:171]: OK, got it. I don't think the proposed wording says that clearly, but as I can't think of a better wording, I'll just make that comment and move on 🙂

Comment (brainwane) [reply pep-0609.rst:171]: How about this wording?

`At the time of writing (June 2020), the Python Steering Council has `standing delegations`_ for currently active packaging interoperability specifications. Anyone in those standing delegations can also be a sponsor for a packaging-related PEP.`

Comment (pradyunsg) [reply pep-0609.rst:171]: Thanks @brainwane for the suggestion! ^>^

I'd like to leave out the specifics of what those delegations mean from this PEP, in favour of pushing for clarification in the standing delegations document instead. I feel the SC standing delegations document would be a much better place to describe what responsibilities / powers have been delegated.

Comment (brainwane) [reply pep-0609.rst:171]: ok! Submitting a PR to that page now.

Comment (brainwane) [reply pep-0609.rst:171]: Submitted https://github.com/pypa/pypa.io/pull/55 and https://github.com/python/steering-council/pull/29 .

**Comment by brainwane** [comment]
Since PEP author @di has approved this PR, could it please be approved and merged? Thanks.

**Comment by pradyunsg** [comment]
@brettcannon A *very* gentle nudge on this.
